index.html

```diff
@@ -25,6 +25,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <nav id="mobile-menu-container">
+        <div class="menu-shape">
+          <img class="img-square" src="./media/svg/square.svg" alt="square" />
+          <i id="mobile-close-menu" class="fa fa-times"></i>
+        </div>
+        <ul>
+          <li><a class="nav-link" href="#">Hello</a></li>
```

Comment:
It seems you added some extra links! I kindly suggest you delete the extra hello `<a>` tag.
It is nice to add a personal touch to your projects! :heart: But sadly in this case we have to follow [this template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
```suggestion

```

Reply:
I have resized the icon and deleted unnecessary link.